java/src/jmri/jmrix/sprog/SprogTrafficController.java

```diff
@@ -284,13 +284,10 @@ public void run() {
             }
             log.debug("Message dequeued id: {}", messageToSend.message.getId());
             // remember who sent this
-//            log.debug("Updating last sender {}");
             lastSender = messageToSend.listener;
             lastId = messageToSend.message.getId();
             // notify all _other_ listeners
-            if (messageToSend.listener != null) {
                 notifyMessage(messageToSend.message, messageToSend.listener);
```

Comment:
This seems to make the indentation wrong for the statement with notifyMessage(), since the if-statement is removed. Removing four spaces in the beginning of line 290 solves it.

Reply:
Sorry about that. Corrected now.

Reply:
Now failing in jmri.jmrit.display.switchboardEditor.SwitchboardEditorTest which I cannot relate to the changes I made.